monkey/infection_monkey/system_info/mimikatz_collector.py

```diff
@@ -49,8 +51,13 @@ def init_mimikatz(self):
             self._get = get_proto(("get", self._dll))
             self._get_text_output_proto = get_text_output_proto(("getTextOutput", self._dll))
             self._isInit = True
+            status = ScanStatus.USED
         except Exception:
             LOG.exception("Error initializing mimikatz collector")
+            status = ScanStatus.SCANNED
```

Comment:
I personally would have done
`status = ScanStatus.USED if self._isInit else ScanStatus.SCANNED`
But that's OK as it is